logstash-core/lib/logstash/instrument/null_metric.rb

```diff
@@ -1,6 +1,16 @@
 # encoding: utf-8
+require "logstash/instrument/metric"
+
 module LogStash module Instrument
  class NullMetric
+   class NullTimedExecution
+     def stop
+     end
+   end
+
+   NULL_TIMED_EXECUTION_INSTANCE = NullTimedExecution.new
```

Comment:
Yes, probably a singleton will be more clear. Change if we feel to do it.

On Fri, 15 Jan 2016 17:04 Pier-Hugues Pellerin <notifications@github.com>
wrote:

> In logstash-core/lib/logstash/instrument/null_metric.rb
> <https://github.com/elastic/logstash/pull/4474#discussion_r49869977>:
>
> >  module LogStash module Instrument
> >   class NullMetric
> > +   class NullTimedExecution
> > +     def stop
> > +     end
> > +   end
> > +
> > +   NULL_TIMED_EXECUTION_INSTANCE = NullTimedExecution.new
>
> both work I guess ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/logstash/pull/4474/files#r49869977>.
>